docs/scarpet/api/Auxiliary.md

```diff
@@ -245,13 +245,26 @@ Example usages:
   // not a problem in apps
 </pre>
 
-### `logger(expr)`
+### `logger(msg), logger(type, msg)`
 
-Prints the message to system logs, and not to chat.
+Prints the message to system logs, and not to chat. By default prints an info, unless you specify otherwise in the `type` parameter.
 
-### read_file(resource, type)
-### delete_file(resource, type)
-### write_file(resource, type, data, ...)
+Available output types:
+
+`'debug'`: white
```

Comment:
there is no need mention colors - every system could have that configured differently.

You also forgot to document the 'air' entity API